src/components/mixins/DataFormat.js

```diff
@@ -0,0 +1,93 @@
+let Validator = require('validatorjs');
+
+// To include another language in the Validator with variable processmaker
+if (window.ProcessMaker && window.ProcessMaker.user && window.ProcessMaker.user.lang) {
+  Validator.useLang(window.ProcessMaker.user.lang);
+}
+
+export default {
+  props: {
+    dataFormat: {
+      type: String,
+      default() {
+        return 'string';
+      },
+    },
+  },
+  data() {
+    return {
+      validatorFormat: null,
+    };
+  },
+  watch: {
+    // Triggered whenever the v-model is updated
+    value() {
+      if (this.$parent.transientData) {
+        this.$parent.transientData[this.name] = this.formatValue(this.value);
+      }
+    },
+  },
+  methods: {
+    formatValueWith(value, format) {
+      this.$set(this, 'dataFormat', format);
+      return this.formatValue(value);
+    },
+    formatValue(value) {
+      if (!value && this.dataFormat !== 'boolean') {
+        return '';
+      }
+
+      //Rules
+      let rules = {
```

Comment:
Use `let` to instantiate variables that will be re-assigned, and use `const` otherwise. In this case, `rules` is not re-assigned so it can be created with `const`.

Reply:
fixed